ci/requirements/docs.yml

```diff
@@ -0,0 +1,26 @@
+name: pygmt
+channels:
+    - conda-forge
+    - defaults
+dependencies:
+    # Required dependencies
+    - pip
+    - gmt=6.3.0
```

Comment:
Also need to update https://github.com/GenericMappingTools/pygmt/blob/main/.github/ISSUE_TEMPLATE/bump_gmt_checklist.md

Reply:
Should be ok to update `bump_gmt_checklist.md` directly in #1990.